templates/interface/myscans2.html

```diff
@@ -254,9 +256,10 @@ <h3 class="panel-title">Starred Scans</h3>
             b = parseInt($(b).text());
             return ((a < b) ? 1 : ((a > b) ?  -1 : 0));
         };
-
+*/
```

Comment:
Yes, new scans at the top of the list is a good idea. 
Would it be possible to remove  ```/*``` and ```*/``` in code, i don't see their purpose (or i might be wrong, explain why you used them) ?